### PR TITLE
add angular integration with tailwind elements

### PR DESCRIPTION
**Pull request by marveluck**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the new behavior?

[resources](https://angular.io/resources) have updated resources links now once this is merged.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by twerske — APPROVED** [review]
lgtm

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -574,6 +574,11 @@
             "desc": "NGX Cookie Service is an Angular Library that can be used to read, set and delete browser cookies",
             "title": "NGX Cookie Service",
             "url": "https://github.com/stevermeister/ngx-cookie-service"
+          },
+          "tailwind-elemetns": {
```
Comment (otodockal) [inline aio/content/marketing/resources.json:578]: typo tailwind-elemetns -> tailwind-elements

**Comment by pkozlowski-opensource** [comment]
Thank you for opening the pull request, we do appreciate all the contributions.

We do follow certain commit message guidelines (to automatically generate changelog, among other things). This PR does not follow the recommended git commit message format and it will need cleanup. You can find more information about our commit message format here: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#-commit-message-format

**Comment by pkozlowski-opensource** [comment]
You will need to sign a CLA as well.

**Comment by marveluck** [comment]
@pkozlowski-opensource CLA signed

**Comment by JeanMeche** [comment]
Hi, you will still need to reword your commit per Pawel's comment !

**Comment by marveluck** [comment]
Sorry it took so long. I have fixed the typo. 

Is there anything else I need to do?

**Comment by JeanMeche** [comment]
Yes, you'll need to rebase on main, squash your commits and reword the remaining commit to follow our [commit message guidelines](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit) !

**Review by twerske — APPROVED** [review]
LGTM (once PR is cleaned up)

**Comment by JeanMeche** [comment]
In absense of cleanup, I'll close it. Feel free to reopen a new PR if you'd like to land those changes !

**Pull request closed without merging**